app/lib/events/event.rb

```diff
@@ -0,0 +1,32 @@
+module Events
+  class Event
+    def initialize
+      @event_hash = {
+        environment: HostingEnvironment.environment_name,
+      }
+    end
+
+    def as_json
+      @event_hash.as_json
```

Comment:
Just noticed that we don't appear to be getting a timestamp in the BigQuery table. I'll recreate the table with that field and let you know it's name so we can add it here.

Reply:
I've just:

- added the optional field `timestamp`. I'm considering making this required, will see what Ravi thinks
- changed `user_id` to be a `BIGNUMERIC`
- renamed the table to `request_events`, just to reduce the duplication of `apply` in the name